config/tech-docs.yml

```diff
@@ -37,3 +37,8 @@ github_repo: moj-analytical-services/user-guidance
 # Slack options
 owner_slack_workspace: asdslack
 default_owner_slack: '#analytical_platform'
+
+footer_links:
+  Get in touch by email: mailto:analytical_platform@digital.justice.gov.uk
+  Slack: https://asdslack.slack.com/messages/C4PF7QAJZ/
```

Comment:
Could we change this to 'Get in touch by Slack' for consistency?

Reply:
👍 done